github/event_types.go

```diff
@@ -891,15 +891,18 @@ type RepositoryVulnerabilityAlertEvent struct {
 
 	//The security alert of the vulnerable dependency.
 	Alert *struct {
-		ID                  *int64     `json:"id,omitempty"`
-		AffectedRange       *string    `json:"affected_range,omitempty"`
-		AffectedPackageName *string    `json:"affected_package_name,omitempty"`
-		ExternalReference   *string    `json:"external_reference,omitempty"`
-		ExternalIdentifier  *string    `json:"external_identifier,omitempty"`
-		FixedIn             *string    `json:"fixed_in,omitempty"`
-		Dismisser           *User      `json:"dismisser,omitempty"`
-		DismissReason       *string    `json:"dismiss_reason,omitempty"`
-		DismissedAt         *Timestamp `json:"dismissed_at,omitempty"`
+		ID                       *int64     `json:"id,omitempty"`
+		AffectedRange            *string    `json:"affected_range,omitempty"`
+		AffectedPackageName      *string    `json:"affected_package_name,omitempty"`
+		ExternalReference        *string    `json:"external_reference,omitempty"`
+		ExternalIdentifier       *string    `json:"external_identifier,omitempty"`
+		GithubSecurityAdvisoryID *string    `json:"ghsa_id,omitempty"`
```

Comment:
```suggestion
		GitHubSecurityAdvisoryID *string    `json:"ghsa_id,omitempty"`
```

Reply:
Apologies for that, thanks for picking up on it